### ISSUE TITLE
Löschen von Kurzlinks?

### ISSUE THREAD

**Issue by sheepyhollow** [PhysikOnline-FFM/gu-urlshorter#16]
Mir ist es nun auch passiert, dass sich das Ziel eines "sprechenden" Kurzlinks verändert hat. 
Mittelfristig muss man sich überlegen, ob es nicht irgendeinen Weg gibt, wie man Shortlinks wieder löschen kann. 

Wie könnte man das ohne User-Verwaltung realisieren?
Löschantrag, der moderiert wird? Einfach nur ein Captcha (so als psychologische Hürde)?

**Comment by tmke8** [comment]
Löschantrag mit Captcha, der moderiert wird, klingt gut. Ich denke nicht, dass das allzu viele werden.

**Comment by svenk** [comment]
Am schönsten fände ich es, wenn man sich (optional) einloggen könnte, um dann seine erstellten Links autonom verwalten (=ändern, löschen) zu können. Ein Script um Löschanträge zu submitten kann man aber auch hinzufügen. In jedem Fall muss das derzeit über den Admin-Account gehen.

**Comment by sheepyhollow** [comment]
Heißt aber, dass ich mich zum Erstellen schon einloggen müsste, wenn ich in Zukunft einen bearbeitbaren Link haben will.

Finde ich auch am besten - macht halt bisschen Arbeit. 

Anbindung per LDAP? Wäre am konsequentesten.

**Comment by svenk** [comment]
Ja, per HRZ-Login. Haben wir bei PhysikOnline schon oft gemacht und geht recht einfach.

Btw, die Diskussion hier ist öffentlich, sheepyhollow ;-)

**Comment by sheepyhollow** [comment]
Ich vergaß.

**Comment by sheepyhollow** [comment]
Da wir es am Dienstag beim Netzwerktreff wieder davon hatten... für Clock müssten wir wohl bald das Fass des Verfahrensverzeichnisses aufmachen. Sollte man das für tinygu.de gleich mit aufmachen, damit man das in der Tasche hat? Selbst, wenn mal ein offizieller SSO kommt, werden wir so oder so ein entsprechendes Papier brauchen.

**Comment by svenk** [comment]
@sheepyhollow, hab deinen Vorschlag zum VVZ nach #23 verschoben.

**Comment by sheepyhollow** [comment]
Bei dudle/doodle kann ich Änderungen etc. auch auf einem niedrigeren Level machen. 
Gut - ist vielleicht nicht gar so kritisch, wie Links. Aber vielleicht kann man sich eine nicht-Nutzerbasierte Lösung ausdenken. Mit einer ID (PW), die ich mir dann halt merken muss oder was Ähnlichem. 
Aber die Option, Poweruser mit Account zu haben, ist nach wie vor nicht schlecht. Viele nutzen das halt auch für 'ne Menge Sachen mittlerweile.

**Comment by svenk** [comment]
Also um mal mit einem Konzept aufzuwarten:
- Einfaches Benutzerinterface wie jetzt für alle normalen User.
- Nach Login (idealerweise mit HRZ-Account) ein erweitertes Benutzerinterface, was folgendes erlaubt:
  - Managen aller selbsterstellten Links (ändern, löschen, usw.)
  - Erstellen von Links mit Ablaufdatum #25

Schon jetzt gibt es die Möglichkeit, zu allen Links Statistiken anzuschauen sowie TinyGU mit eigenen Domains zu betreiben (per DNS: `go.meininstitut.de -> CNAME tinygu.de`).
